packages/elements/src/components/FileInput/index.tsx

```diff
@@ -32,6 +33,7 @@ export const FileInput = ({
   testProps,
   inputProps,
   required = false,
+  onFilenameClick = undefined,
```

Comment:
`onFilenameClick?: () => void` `?` ensures it will be undefined by default, no need to set a default param of undefined